src/Public/OpenApi.ps1

```diff
@@ -845,9 +855,9 @@ function Set-PodeOARequest {
                 }
 
                 if ($null -ne $RequestBody) {
-                    # Only 'POST', 'PUT', 'PATCH' can have a request body
-                    if (('POST', 'PUT', 'PATCH') -inotcontains $r.Method ) {
-                        # {0} operations cannot have a Request Body.
+                    # Check if AllowNonStandardBody is used or if the method is typically allowed to have a body
+                    if (-not $AllowNonStandardBody -and ('POST', 'PUT', 'PATCH') -inotcontains $r.Method) {
```

Comment:
`!` instead of `-not` :)

Reply:
Always I forget😊
With my other projects I’m using -not